source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRActionSetRenderer.mm

```diff
@@ -110,4 +112,14 @@ - (UIView *)renderButtons:(ACRView *)rootView
     return containingView;
 }
 
++ (ACRCardElementType)elemType {
+    return ACRActionSet;
+}
+
+- (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup rootView:(ACRView *)rootView inputs:(NSArray *)inputs baseCardElement:(ACOBaseCardElement *)acoElem hostConfig:(ACOHostConfig *)acoConfig {
+        ACOAdaptiveCard *card = [[ACOAdaptiveCard alloc] init];
+        [card setCard:[[rootView card] card]];
```

Comment:
Fix indentation

Reply:
thanks

---
In reply to: [270518627](https://github.com/Microsoft/AdaptiveCards/pull/2582#discussion_r270518627) [](ancestors = 270518627)